src/features/recipes/components/RecipeDetails/RecipeDetails.jsx

```diff
@@ -1,45 +1,226 @@
+      const isPlural = quantity > 1 ? true : false;
+      console.log(isPlural);
+      const measurement = (
+        <span className="measurement text-tangerine font-bold font-open-sans">
+          <span className="quantity">
+            {quantity && quantity !== "0" ? quantity + " " : "-"}
+          </span>
+          <span className="unit">
+            {isPlural ? unit.display_plural + " " : unit.display_singular + " "}
+          </span>
+        </span>
+      );
+      const name =
+        isPlural && unit.display_plural === ""
+          ? component.ingredient.display_plural + " "
+          : component.ingredient.display_singular + " ";
+      const note = component.extra_comment
+        ? ` (${component.extra_comment})`
+        : "";
       return (
-        <div key={key} className="w-full">
-          <strong>{key}: </strong>
-          {JSON.stringify(value).slice(0, 80)}
-        </div>
+        <li key={index}>
+          {measurement}
+          {name}
+          {note}
+        </li>
       );
     });
+    return (
+      <div className="recipe-component py-2">
+        <Heading level="h4" variant="lava">
+          {name || "You'll Need:"}
+        </Heading>
+        <ul>{components}</ul>
+      </div>
+    );
+  };
+
+  const IngredientSections = ({ sections }) => {
+    return (
+      <div>
+        <Heading level="h2" variant="tangerine" id="ingredients">
+          Ingredients
+        </Heading>
+        {sections.map((section, index) => {
+          return (
+            <Section
+              key={index}
+              list={section.components}
+              name={section.name}
+            />
+          );
+        })}
+      </div>
+    );
+  };
 
   return (
-    <div className="p-10">
-      <Heading level="h1" variant="tangerine">
-        {recipe.name}
-      </Heading>
-      <img
-        src={recipe.thumbnail_url}
-        alt={recipe.name}
-        className="w-[400px] h-[400px] object-cover rounded-full"
-      />
-      {recipeDetails}
+    <div className="page-wrapper bg-sky">
+      <div className="title-wrapper bg-title-cutout bg-cover bg-no-repeat p-4 pb-12">
+        <Heading level="h1" variant="lava" className="pl-6 pt-6 leading-[1]">
+          {recipe.name}
+        </Heading>
+        <hr className="mb-1 mt-3 h-2 w-[40%] rounded-full border-none bg-gradient-tangerine-diagonal" />
+      </div>
+      <div className="p-10">
+        <Card className="intro-card">
+          <div>
+            <p>
+              <Description description={recipe.description} />
+            </p>
+          </div>
+          <img
+            src={recipe.thumbnail_url}
+            alt={recipe.name}
+            className="h-[270px] w-[270px] rounded-full object-cover rotate-12"
+          />
+        </Card>
+        <Card className="quick-links-card">
+          <Heading level="h4" variant="lava">
+            Filed Under
+          </Heading>
+          <Topics topics={recipe.topics} />
```

Comment:
Maybe replace this with a logical (&&) operator like so:
```suggestion
         { recipe.topics && <Topics topics={recipe.topics} />}
```
this will only render the Topics component when `recipe.topics` equates to true otherwise anything after the `&&` will not render.